src/scss/components/switch/_index.scss

```diff
@@ -71,6 +76,7 @@ hx-switch-control > input[type="checkbox"] ~ label {
 hx-switch-control > input[type="checkbox"]:checked ~ label {
   > hx-switch {
     background-color: $cyan-900;
+    background-color: var(--hxSwitchControl-switch-checked-bgcolor, $cyan-900);
```

Comment:
Please use updated colors to replace `$cyan-XXX`.

Reply:
Changes have been done, please review.